packages/menu/src/Menu.tsx

```diff
@@ -388,12 +388,18 @@ function Menu({
 
 Menu.displayName = 'Menu';
 
+// __TARGET__ is a global variable that indicates the webpack build target.
+//
+// We're typing this here because doing it globally was proving problematic.
+// We should solve for this if we need to use __TARGET__ elsewhere.
```

Comment:
Should we type this in lib and have places import. I think this will be the third usage in our repo

Reply:
Didn't even see that it was used everywhere in this PR -- maybe now is a good time to add to lib?

Reply:
I actually think that may break the build. This is one of those things that `import type` might actually be good for. It's worth testing, but trying to import a constant that's initialized without a value might overwrite the build variable.

Reply:
Yep, so if you try to add it to `lib`, it outputs:

```js
current:"web"===a.__TARGET__?u.default.instanceOf(Element):u.default.any
```

The problem with this is that the bundle doesn't have an instance of the `__TARGET__` variable, and will default to using the `node` environment. If we keep it as it's done in the PR, any code in a conditional that relies on the value of `__TARGET__` is removed altogether when appropriate.

Something interesting I realized is that now that we're outputting ES Modules as well, the ES Modules will output the `__TARGET__` usage directly. I might need to look into this, though I'd suggest this not be a blocker for the PR since we:
1. Don't truly support ES Modules as an API yet.
2. The emotion package already outputs this, and it hasn't yet been problematic.

I will make a ticket for the issue though.

Reply:
👍